packages/database/src/utilities/migration.js

```diff
@@ -15,7 +15,12 @@ class RequiredParameterError extends Error {
 }
 
 export const insertObject = async (db, table, data, onError) =>
-  db.insert(table, Object.keys(data), Object.values(data), onError);
+  db.insert(
+    table,
+    Object.keys(data),
+    Object.values(data).map(x => (Array.isArray(x) ? `{${arrayToDoubleQuotedDbString(x)}}` : x)),
```

Comment:
Please check no previous migration is passing an array in (maybe that's automatically checked because it just wouldn't have worked)

Reply:
So looking back at this there _are_ a couple of situations where we pass an array in previous migrations. These are situations where the database column is a `jsonb` type, and therefore the stringified `[ x, y ,z ]` format is valid. But that format of course isn't valid for a column that is actually an array type `something[]` and that's when we need to convert to `{ x, y, z }` format.

This was a really good catch that I absolutely didn't consider. I think the best option is to just revert this change and run the `arrayToDoubleQuotedDbString` conversion in my individual migration.

Reply:
Sounds good, thanks for checking it out!